### DISCUSSION TITLE
ISSUE_TEMPLATES or PULL_REQUEST_TEMPLATE in .github/.github for organization are not working

### DISCUSSION THREAD

**Discussion by sindhupavanimovva** [community/community#29764] [Projects and Issues]
What is the current behaviour?
We don't get default templates in our repositories. This is possible because we've previously used individual repo templates.

What is the expected behaviour?
When we open a GitHub issue or PR, each repository should be pre-filled from our templates.

**Comment by lucasrabaca** [comment]
in my organization it is not working for private repositories, only the public ones.
